### ISSUE TITLE
Maintainer wanted

### ISSUE THREAD

**Issue by Zenoo** [amsul/pickadate.js#1233]
Hi,  

I saw this project on the [Maintainers wanted](https://github.com/pickhardt/maintainers-wanted) list.  
Are you still looking for a maintainer @amsul ?

**Comment by amsul** [comment]
Yup. @DanielRuf has primarily been helping out and I'm there's a ton of places to help out with!

**Comment by Zenoo** [comment]
Alright, I'll start by opening a few PRs to fix existing issues soon then.

**Comment by LucidSamuel** [comment]
> Yup. @DanielRuf has primarily been helping out and I'm there's a ton of places to help out with!

Hello just checking in to see if you still need maintainers on this project, would love to help fix issues and contribute to the project.

**Comment by DanielRuf** [comment]
> Hello just checking in to see if you still need maintainers on this project, would love to help fix issues and contribute to the project.

Yes, any help is very welcome and so are more / new maintainers.
